pkg/kopscodecs/codecs_test.go

```diff
@@ -76,6 +76,41 @@ func TestToVersionedYaml(t *testing.T) {
 
 }
 
+func TestToVersionedJSON(t *testing.T) {
+	grid := []struct {
+		obj      runtime.Object
+		expected string
+	}{
+		{
+			obj: &v1alpha2.Cluster{
+				ObjectMeta: metav1.ObjectMeta{
+					CreationTimestamp: testTimestamp,
+					Name:              "hello",
+				},
+				Spec: v1alpha2.ClusterSpec{
+					KubernetesVersion: "1.2.3",
+				},
+			},
+			expected: heredoc.Doc(`{"kind":"Cluster","apiVersion":"kops.k8s.io/v1alpha2","metadata":{"name":"hello","creationTimestamp":"2017-01-01T00:00:00Z"},"spec":{"kubernetesVersion":"1.2.3"}}`),
```

Comment:
I don't think you need to use `heredoc.Doc` here. In the test above, the json is formatted, while here it is not

Reply:
Thanks, done.